AIDungeon_2.ipynb

```diff
@@ -45,7 +45,7 @@
         "colab": {}
       },
       "source": [
-        "!git clone https://github.com/nickwalton/AIDungeon/\n",
+        "!git clone --depth 1 https://github.com/nickwalton/AIDungeon/\n",
```

Comment:
+1 Why not include `--branch master`?

Reply:
Thanks for the suggestion @ben-bay. Added that.